components/core/src/clp/clg/CMakeLists.txt

```diff
@@ -1,5 +1,12 @@
 set(
         CLG_SOURCES
+        ../GlobalMetadataDB.hpp
```

Comment:
Is there a rule/justification on when to use relative path

Reply:
There isn't a rule right now. I was using relative paths for things *in* the clp namespace and absolute paths for things outside the clp namespace.